### PR TITLE
Update to NG7

### PR DESCRIPTION
**Pull request by ChampionAsh5357**
Supersedes #54 

Updates the documentation to reflect the changes made in NG7.

------------------
Preview URL: https://pr-68.neoforged-docs-previews.pages.dev

### CODE REVIEW

**Comment by marchermans** [comment]
image
Typo in the version.

**Comment by ChampionAsh5357** [comment]
Fixed hopefully.

File: neogradle/docs/configuration/index.mdx [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-# ForgeGradle Configurations
+# NeoGradle Configurations
 
-ForgeGradle has numerous configurations that can change how the development environment is configured. Most configurations are set using the `minecraft` block; however, some others can be specified within the `dependencies` block or modify the built `jar`, such as `reobfJar`.
+NeoGradle has numerous configurations that can change how the development environment is configured.
 
 ## Enabling Access Transformers
 
-[Access Transformers][at] can widen the visibility or modify the `final` flag of Minecraft classes, methods, and fields.
+Access Transformers can widen the visibility or modify the `final` flag of Minecraft classes, methods, and fields.
```
Comment (IchHabeHunger54) [inline neogradle/docs/configuration/index.mdx:10]: Link to AT article.

File: neogradle/docs/configuration/runs.md [hunk 2]
```diff
@@ -1,20 +1,16 @@
 # Run Configurations
 
-Run configurations define how an instance of the game is going to run. This includes arguments, working directories, task names, etc. Run configurations are defined within the `minecraft.runs` block. While no runs are configured by default, [Forge][userdev] does provide the configurations `client`, `server`, `data`, or `gameTestServer`.
+Run configurations define how an instance of the game is going to run. This includes arguments, working directories, task names, etc. Run configurations are defined within the `runs` block. While no runs are configured by default, NeoGradle does provide the configurations `client`, `server`, `data`, or `gameTestServer`.
```
Comment (IchHabeHunger54) [inline neogradle/docs/configuration/runs.md:3]:
```suggestion
Run configurations define how an instance of the game is going to run. This includes arguments, working directories, task names, etc. Run configurations are defined within the `runs` block. No runs are configured by default, however NeoGradle provides the configurations `client`, `server`, `data` and `gameTestServer` out of the box.
```

File: neogradle/docs/index.md [hunk 3]
```diff
@@ -4,50 +4,66 @@ sidebar_position: 0
...
     // ...
 }
 ```
 
-[ForgeGradle]: https://github.com/MinecraftForge/ForgeGradle
+:::note
+While you can use version ranges for the NeoGradle plugin, it is not recommended to do so, as that may lead to more frequent decompilation and recompilation rounds and possible behavioral changes. You can find the latest NeoGradle version on our [Project Listing][gradlelisting].
+:::
+
+While you can use version ranges for the NeoGradle plugin, it is not recommended to do so, as that may lead to more frequent decompilation and recompilation rounds and possible behavioral changes.
```
Comment (IchHabeHunger54) [inline neogradle/docs/index.md:43]: Copypasta?

File: neogradle/docs/parchment.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
+# Parchment
+
+[Parchment] is a community-sourced set of mappings of *parameter names* and *javadocs* that augment the official names released by Mojang. By using parchment, you can get parameter names for most Minecraft methods, instead of the indescriptive `p_` names.
```
Comment (IchHabeHunger54) [inline neogradle/docs/parchment.md:3]: Not a fan of the cursive, consider using bold (**) instead.

**Review by IchHabeHunger54 — CHANGES_REQUESTED** [review]
A few rephrasings, otherwise looking good. I am not familiar with NeoGradle in any way, this was purely a "does this sound like English?" kind of review.

It might be worth explaining what Parchment does in a bit more detail. Most notably, it should be pointed out that applying Parchment to an existing project will not cause problems because parameter names (and javadocs) are ignored by the JVM, and will thus only impact development after it has been included in the project. A link to the Parchment contribution page might also be a good idea.

**Review by IchHabeHunger54 — APPROVED** [review]

**Pull request merged**